### PR TITLE
Add information quarkus.http.limits.max-form-attribute-size

### PR DESCRIPTION
**Pull request by technical-debt-collector**
Providing some documentation about the parameter as suggested by @geoand in https://github.com/quarkusio/quarkus/issues/16422

### CODE REVIEW

**Review by geoand — COMMENTED** [review]
Thanks for this.

I think it would be best if this were a note, and also not the first thing a user reads in the multipart section.

**Comment by technical-debt-collector** [comment]
> I think it would be best if this were a note, and also not the first thing a user reads in the multipart section.
I'm assuming you only refer to the changes in resteasy-reactive-migration.adoc - sure, I can move it down one paragraph and pack it into a note if you think that is better.
Let me know if I misunderstood you.

File: docs/src/main/asciidoc/resteasy-reactive-migration.adoc [hunk 1]
```diff
@@ -104,9 +104,11 @@ RESTEasy Reactive does not support this feature. Instead, users that have provid
 using one of the methods described in the xref:cdi-reference.adoc#bean_discovery[Bean Discovery] section of the CDI guide.
 
 === Multipart support
-
 HTTP Multipart support in RESTEasy Reactive does **not** reuse the same types or annotations as RESTEasy Classic and thus users are encouraged to read <<resteasy-reactive#multipart,this>> part of the reference documentation.
 
+NOTE: Users migrating multipart resources to RESTEasy Reactive should be aware of the configuration parameter `quarkus.http.limits.max-form-attribute-size`, as this poses an upper limit to the size of each part.
```
Comment (geoand) [inline docs/src/main/asciidoc/resteasy-reactive-migration.adoc:109]: I see no reason to be repeating the information. I would drop this part and make the other part you added a note

**Review by geoand — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/resteasy-reactive-migration.adoc (hunk 1 above)
Comment (technical-debt-collector) [reply docs/src/main/asciidoc/resteasy-reactive-migration.adoc:109]: This behaviour and parameter is completely new to users migrating from Classic to Reactive. Shouldn't the migration page mention anything about this at all?

Comment (geoand) [reply docs/src/main/asciidoc/resteasy-reactive-migration.adoc:109]: Sorry, I had not noticed the doc that was changed. My bad

**Review by geoand — APPROVED** [review]

**Comment by geoand** [comment]
Thanks for this!

**Pull request merged**